lib/handlerTemplate.js

```diff
@@ -6,9 +6,10 @@
  */
 
 var P = require('bluebird');
-var URI = require('swagger-router').URI;
+var swaggerRouter = require('swagger-router');
+var URI = swaggerRouter.URI;
```

Comment:
Minor nit: let's place these two variables *after* all of the requires. I feel it increases readability.